sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -315,6 +323,23 @@ public partial class BlobAppendInfo
         public bool IsServerEncrypted { get { throw null; } }
         public System.DateTimeOffset LastModified { get { throw null; } }
     }
+    public partial class BlobBaseDownloadOptions
```

Comment:
Arch board might want us to name this `BlobDownloadOptions`, I don't think we currently have any model classes with `Base` in the name.

Reply:
This name was present in an APIView they looked at.

@KrzysztofCwalina do you have a preference on this? The client name is `BlobBaseClient`, so this name does match current patterns, but I can understand an exception for this case.

Reply:
+1 for `BlobDownloadOptions`